packages/wrangler/README.md

```diff
@@ -43,7 +43,7 @@ $ npm install wrangler --save-dev
 
 ## Configuration:
 
-Wrangler is configured via a `wrangler.toml` file in the project root. When utilizing the `wrangler init` command, a `wrangler.toml` file will be created for you.
+Wrangler is configured via a `wrangler.toml` or `wrangler.json` file in the project root. When utilizing the `wrangler init` command, a `wrangler.toml` file will be created for you.
```

Comment:
I haven't changed the `wrangler init` path since it delegates to C3 (and the non-delegated version is deprecated). I'll change C3 in a month or so once this has had time to settle